lib/readme.txt

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
Great job including a README. Typically, you'll want to have this be formatted in markdown.